source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -7,28 +7,42 @@
 
 [WARNING]
 ====
-Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
+Do not let certificates expire. If they expire, the host and {hypervisor-shortname} stop responding, and recovery is an error-prone and time-consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
 ====
 
 .Procedure
 
-. In the Administration Portal, click menu:Compute[Hosts].
-. Migrate the virtual machines away from the host.
-. Click menu:Management[Maintenance] and btn:[OK].
-. Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
-. Once enrollment is complete, click menu:Management[Activate].
+. Renew the host certificates:
+.. In the Administration Portal, click menu:Compute[Hosts].
+.. Click menu:Management[Maintenance] and then click btn:[OK].
+The virtual machines should automatically migrate away from the host. If they are pinned or otherwise non-migratable, you need to shut them down.
+.. Once the host is in maintenance mode, and there are no more virtual machines remaining on this host, click menu:Installation[Enroll Certificate].
+.. Once enrollment is complete, click menu:Management[Activate].
 
-.Next steps
-You must also renew Engine certificates.
-
-. Run `engine-setup` from an SSH or console session on the {engine-appliance-name} system. The `engine-setup` script prompts you with some configuration questions.
+. Renew the {engine-name} certificates:
+.. For a self-hosted engine, log in to the host and put it in global maintenance mode:
 +
 ----
-# engine-setup
+# hosted-engine --set-maintenance --mode=global
 ----
+.. For a self-hosted engine and standalone {engine-name}, log in to the {engine-name} and run `engine-setup`:
 +
-. Respond to the questions as appropriate or use an answers file.
-
+----
+# engine-setup --offline
+----
++
+The `engine-setup` script prompts you with configuration questions.
```

Comment:
I am not sure you need this text. I think we can assume that the user has run engine-setup in the past or knows how to use an answer file.

Reply:
we use it everywhere, I think we should leave it just in case it helps orient or clarify for the user. 
Having gone through these installation and engine-setup procedures hundreds of times myself, I can tell you, it is still easy to get disoriented and confused...